packages/block-library/src/query-pagination-numbers/block.json

```diff
@@ -10,7 +10,26 @@
 	"usesContext": [ "queryId", "query" ],
 	"supports": {
 		"reusable": false,
-		"html": false
+		"html": false,
+		"color": {
+			"gradients": true,
+			"text": false
```

Comment:
To answer both these questions, I used the same settings as the query pagination next and previous page inner blocks, for consistency.
We can add padding, but it would need to be added to the other two blocks as well, in my opinion. 
The text and link color settings are on the parent "query pagination" block. (which does not have any text of its own...)

Reply:
Optionally, if I need to remove blockgap, place the padding on the numbers and dots, but then there won't be padding around the container, the set of numbers, where the background is applied.

Reply:
> I used the same settings as the query pagination next and previous page inner blocks, for consistency.

Ah, good point, I see now. Thanks! I think it'd be good for all three to be able to adjust padding and individual color controls, but that could totally be something we explore in a different PR. For now, going with consistency with the other blocks sounds good to me.

Reply:
>Is there a reason we don't want folks to also be able to adjust the text color?

Actually there is. These two blocks `prev/next` have no wrapper and are always a link(`a element`), but the way color supports work is to apply css rules for the children elements. For example if we added the `link color support` would try to target innner `a` elements from the block, which do not exist 😄 

Now we have kept the color support to the parent `Query Pagination` block which can contain both links and text(in Pagination Numbers) and users can control this in one place.